design-system.html

```diff
@@ -0,0 +1,249 @@
+                      <h2>Similar components</h2>
+
+                      <ul class="list">
+                        {{#each similar}}
+                        <li><a href="/design-patterns/patterns/components/{{ url }}">{{ title }}</a></li>
+                        {{/each}}
+                      </ul>
+                    </div>
+                    {{/if}}
+                    {{/if}}
+
+                    {{#if isPattern}}
+                    {{#if uses}}
+                    <div class="column-one-quarter related-content-grid-item">
+                      <h2>Uses components</h2>
+
+                      <ul class="list">
+                        {{#each uses}}
+                        <li><a href="/design-patterns/patterns/components/{{ url }}">{{ title }}</a></li>
+                        {{/each}}
+                      </ul>
+                    </div>
+                    {{/if}}
+                    {{/if}}
+
+                    <div class="column-one-quarter related-content-grid-item">
+                      <h2>Further resources</h2>
+
+                      <ul class="list">
+                        {{#if seprepo}}
+                        <li><a href="">{{ title }} repository on GitHub</a></li>
+                        {{/if}}
+
+                        {{!--
+                        TODO: make these links dynamic with a link_to helper
+                        <li>{{{link_to 'Edit this guidance on GitHub' edit_link}}}</li>
+                        <li>{{{link_to 'Raise an issue on GitHub' (issues_link title)}}}</li>
+                        <li>{{{link_to 'History of this page' history_link}}}</li>
+                        --}}
+
+                        <li><a href="#">Edit this guidance on GitHub</a></li>
+                        <li><a href="#">Raise an issue on GitHub</a></li>
+                        <li><a href="#">History of this page</a></li>
+                      </ul>
+                    </div>
+
+                    <div class="column-one-quarter related-content-grid-item">
+                      <dl class="meta-data-bottom">
+                        {{#if aliases}}
+                        <dt>Also known as:</dt>
+                        <dd>{{ aliases }}</dd>
+                        {{/if}}
+                        <dt>Last reviewed:</dt>
+                        <dd>{{#if date}}{{ date }}{{else}}30 March 2017{{/if}}</dd>
+                      </dl>
+                    </div>
+                  </div>
+                </div>
+
+                <div class="callout">
+                  <h2 class="heading-medium">Get in touch</h2>
+                  <p>If you’ve got a question, idea or suggestion share it in <a href="https://ukgovernmentdigital.slack.com/messages/govuk-design-system">#govuk-design-system</a> on cross-government Slack (<a href="slack://channel?team=T04V6EBTR&amp;id=C6DMEH5R6">open in app</a>) or <a href="#">email the Design System team</a></p>
```

Comment:
Because this is the HMRC system at the moment I think we should point to HMRC slack in the SDT channel